tao_compiler/mlir/disc/tools/disc-pdll/tests/single_output.pdll

```diff
@@ -0,0 +1,39 @@
+/// Pre-defined custom call prototypes
+///
+/// const std::string kDefaultHelperFunctionDeclarations = R"pdll(
+///   Rewrite PackValue_1(tag : Attr, v0 : Value) -> ValueRange;
+///   Rewrite PackValue_2(tag : Attr, v0 : Value, v1 : Value) -> ValueRange;
+///   Rewrite UnpackValue_1(v : ValueRange) -> (Value);
+///   Rewrite UnpackValue_2(v : ValueRange) -> (Value, Value);
+///   Rewrite CreateCustomCall(tag : Attr, inputs : ValueRange, outputs : ValueRange) -> (op: Op, new_outputs : ValueRange);
+///   Rewrite SetAttr(op : Op, key : Attr, value : Attr);
+///   Rewrite SetCustomAttr(op : Op, key : Attr, value : Attr);
+/// )pdll";
+
+
+// A new Pattern
+Pattern TFFusedConvRelu {
+  /// match phase: define the pattern
+  let data_format_attr : Attr;
+  let conv = op<tf.Conv2D>(input : Value, weights : Value) { data_format = data_format_attr };
+  let relu = op<tf.Relu>(conv.0);
+
+  /// rewrite phase
+  rewrite relu with {
+    /// 1. create custom call op
+    let inputs = PackValue_2(attr<"\"in\"">, input, weights);
+    let outputs = PackValue_1(attr<"\"out\"">, relu.0);
+    let infos = CreateCustomCall(attr<"\"op\"">, inputs, outputs);
```

Comment:
It seems that attribute `in` / `out` / `op` doesn't help to express the rewrite rule?

Reply:
see the above comment.